Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Driver/Controls/ScrollingTextArea.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Meadow.Foundation.Graphics.MicroLayout;
+
+/// <summary>
+/// Represents an auto-scrolling text area control in the user interface.
+/// </summary>
+public class ScrollingTextArea : ThemedControl
+{
+    public ScrollingTextArea(int left, int top, int width, int height) : base(left, top, width, height)
+    {
+    }
+
+    public override void ApplyTheme(DisplayTheme theme)
```

Comment:
xml

Reply:
Bah, this wasn't even supposed to be in the PR, it's in-progress